examples/large_models/trt_llm/llama/README.md

```diff
@@ -10,6 +10,7 @@ This will downgrade the versions of PyTorch & Triton but this doesn't cause any
 
 ```
 pip install tensorrt_llm==0.10.0 --extra-index-url https://pypi.nvidia.com
+pip install tensorrt-cu12==10.1.0
```

Comment:
Is this CUDA 12 exclusive? In that case we should inform people to install torch with CUDA 12 as well.

Reply:
It doesn't mention it but all their docs are pointing to CUDA 12.x . Let me mention that its tested with CUDA 12.1